otelx/otel_test.go

```diff
@@ -0,0 +1,53 @@
+package otelx
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/ory/x/logrusx"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func TestJaegerTracer(t *testing.T) {
+	host := "127.0.0.1"
+	port := "6831"
```

Comment:
Maybe use `freeport.Port` or `0` to get a free port.
If you use port 0 here, you can get the actual port from the socket.